src/components/results/results.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import DoctorInfoList from './doctorInfoList.js';
+import TotalResults from './totalResults.js';
+
+
+
+class Results extends React.Component {
+
+    constructor(props) {
+      super(props);
+      this.state = {      
+          data: this.props.resultsData.results  
```

Comment:
[comment] We can address this in more detail as you progress, but just calling something "data" is vague and will make it difficult to understand your state as your application grows in complexity.